test/fixtures/runtime-hooks/nuxt.config.ts

```diff
@@ -0,0 +1,21 @@
+import MyModule from '../../../src/module'
+
+export default defineNuxtConfig({
+  modules: [
+    MyModule
+  ],
+  routeRules:{
+    '/test': {
+      headers: {
+        'x-xss-protection': '1',
+      }
+    }
+  },
+  security: {
+    nonce: false,
+    runtimeHooks: true,
+    headers: {
+      contentSecurityPolicy: false
```

Comment:
@huang-julien is it required to turn off nonces and content security policy headers here?

When I keep `contentSecurityPolicy` headers enabled here, the headers set in the hook are not applied it seems.
When I disable `contentSecurityPolicy` headers here, the headers set in the hook are applied but without `{{nonce}}` replacement. Is that intended?